query/src/storages/system/users_table.rs

```diff
@@ -52,12 +52,22 @@ impl AsyncSystemTable for UsersTable {
             .iter()
             .map(|x| x.auth_info.get_auth_string())
             .collect();
+        let default_roles: Vec<String> = users
+            .iter()
+            .map(|x| {
+                x.option
+                    .default_role()
+                    .clone()
+                    .unwrap_or_else(|| "".to_string())
```

Comment:
unwrap_or_default is enough?

Reply:
oh, I did `unwrap_or_default` at first, but clippy suggest me to change into `unwrap_or_else` instead